tests/dynamic_data/publications/layer_external_db/external_db_test.py

```diff
@@ -175,6 +175,24 @@
         'exp_languages': set(),
         'exp_scale_denominator': None,  # Layman doesn't guess scale denominator for (multi)points
     },
+    'point_epsg_3034': {
+        'file_path': "tests/dynamic_data/publications/crs/vectors/sample_point_cz_3034.shp",
+        'style_file': 'tests/dynamic_data/publications/crs/vectors/sample_point_cz.sld',
+        'schema_name': 'public',
+        'table_name': 'point_3034',
+        'primary_key_column': 'ogc_fid',
+        'geo_column_name': 'wkb_geometry',
+        'db_connection_str': None,
+        'additional_geo_column': None,
+        'exp_thumbnail': 'tests/dynamic_data/publications/crs/vectors/sample_point_cz_3034_thumbnail.png',
```

Comment:
Why 3034 works now?

Reply:
Actually it probably works in the same way as before.

The problem with 3034 is computation of bounding box. Default PostGIS 3034 definition gets wrong bounding box, so we create our own definition of 3034 in internal database to get right bounding box from PostGIS.

Data in WMS are transformed using GeoServer with its own 3034 definition. So thumbnail of external table in 3034 should be correct anyway.

Bounding box is also correct, because we transform bounding boxes in internal database, not in external database.